Steam Web Integration.user.js

```diff
@@ -38,10 +39,12 @@ const ownedIcon = "&#10004;"; // HTML entity code for '✔' (default).
 const ownedColor = "green"; // Color of the icon for owned apps and subs.
 const unownedIcon = "&#10008;"; // HTML entity code for '✘' (default).
 const unownedColor = "red"; // Color of the icon for unowned apps and subs.
-const decommissionedIcon = "&#128465;"; // HTML entity code for '🗑' (default).
+const decommissionedIcon = "&#128465;"; // HTML entity code for '🗑' (default). ☠ ("&#9760;") is a good choice if this symbol is unavailable.
 const decommissionedColor = "initial"; // Color of the icon for removed or delisted apps and subs.
-const cardIcon = "&#x1F0A1"; // HTML entity code for '🂡' (default).
+const cardIcon = "&#x1F0A1"; // HTML entity code for '🂡' (default). ♠ ("&spades;") is an alternative if this symbol is unavailable.
 const cardColor = "blue"; // Color of the icon for cards.
+const bundleIcon = "&#127873;"; // HTML entity code for '🎁' (default). Barter.vg uses ✽ ("&#10045;").
+const bundleColor = "initial"; // Color of the icon for bundled apps.
```

Comment:
in FF and Chrome that gift thing didn't react to being coloured. the idea was to set it to `yellow` or `orange`! :1st_place_medal: